vulnerabilities/data_source.py

```diff
@@ -56,6 +56,21 @@ def __post_init__(self):
         if not any([self.url, self.reference_id]):
             raise TypeError
 
+@dataclasses.dataclass
+class VulnPkgVersionRange:
```

Comment:
Could we find a better name?
PackageSpecification may be? what else could be a good one?